src/modules/queue/QueueComponent.jsx

```diff
@@ -103,50 +82,14 @@ const QueueButton = styled.button`
 	border-radius: 50%;
 	box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.1);
 	cursor: pointer;
-	display: inline-block;
-	height: 64px;
-	margin-top: 16px;
+	height: 4rem;
+	width: 4rem;
+	margin-top: 1rem;
 	outline: none;
-	position: relative;
-	width: 64px;
-	z-index: 2;
-
-	image {
-		height: 20px;
-		width: 20px;
-		opacity: 1;
-		transform: transition(all 0.1s);
-	}
 
-	.queue-x {
-		height: 20px;
-		left: 50%;
-		opacity: 0;
+	&.open {
```

Comment:
Notice this is the only place where I'm applying conditional CSS classes, and it's just to make sure the button appears in the same place when it's open and closed to keep parity with your version. 

Normally I wouldn't even do that, and then I'd have no logic in my CSS.